packages/ocaml-variants/ocaml-variants.5.1.1+flambda2/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+version: "5.1.1+flambda2"
+synopsis: "Jane Street compiler variant with Flambda2 backend"
+depends: [
+  "ocaml" {= "5.1.1" & post}
+  "base-unix" {post}
+  "base-bigarray" {post}
+  "base-threads" {post}
+  "base-flambda2" {post}
+  "dune-secondary" {="3.8.1"}
+  "menhir-secondary"
```

Comment:
menhir-secondary (as dune-secondary) is quite old - is there a reason it's not therefore constrained here? (curiosity more than anything else)

Reply:
I'm deferring to downstream here, as those are the versions they are using for their bootstrap.

Reply:
I think menhir versions need to stay as-is, at least for the moment.